repos/system_upgrade/el7toel8/libraries/overlaygen.py

```diff
@@ -7,6 +7,61 @@
 from leapp.libraries.stdlib import CalledProcessError, api, run
 
 
+def _ensure_enough_diskimage_space(space_needed, directory):
+    stat = os.statvfs(directory)
+    if (stat.f_frsize * stat.f_bavail) < (space_needed * 1024 * 1024):
+        message = 'Not enough space available for creating required disk images. Needed: {} MiB'.format(space_needed)
```

Comment:
I would suggest to show users where we are detecting not enough space. I know we show in the logs we are creating "/var/lib/leapp/scratch" but it may not be clear from the first look...